pkg/controller/installation/products/codeready/reconciler.go

```diff
@@ -128,8 +129,8 @@ func (r *Reconciler) Reconcile(ctx context.Context, instance *integreatlyv1alpha
 		return phase, err
 	}
 
-	phase, err = r.reconcileBackups(ctx, serverClient, namespace)
-	if err != nil || phase != integreatlyv1alpha1.PhaseCompleted {
+	phase, err = r.reconcileBackups(ctx, serverClient)
+	if err != nil || phase != v1alpha1.PhaseCompleted {
```

Comment:
I think we're calling this integreatlyv1alpha1 everywhere now, to keep the import names consistent.

Reply:
Good catch. I think I messed it up when rebasing 🙈 
I pushed a fix now.